instruments/AWGBase.py

```diff
@@ -0,0 +1,86 @@
+"""
+AWGs
```

Comment:
I would expand the comments here with instructions of how to create your own driver.